src/components/common/Dialogs/AwardAndSmiteDialogs/ManageReputationContainer/validation.ts

```diff
@@ -0,0 +1,49 @@
+import Decimal from 'decimal.js';
+import { boolean, number, object, string } from 'yup';
+
+import { intl } from '~utils/intl';
+
+const { formatMessage } = intl({
+  amountZero: 'Amount must be greater than zero',
+  maxAmount: "Amount must be less than the user's reputation",
+});
```

Comment:
These messages don't actually show up so they're a bit pointless (maybe useful for devs revisiting this in the future? I actually found it confusing because it made me think the validation was broken). I am tempted to remove and just pass empty strings into the test function.

Reply:
I'm up for removing any dead code